src/test/java/io/github/syst3ms/skriptparser/parsing/SyntaxParserTest.java

```diff
@@ -101,7 +101,9 @@ private static File[] getResourceFolderFiles(String folder) {
         URL url = ClassLoader.getSystemResource(folder);
         if (url != null) {
             var path = url.getPath();
-            return new File(path).listFiles();
+            File[] files = new File(path).listFiles();
```

Comment:
```suggestion
            var files = new File(path).listFiles();
```
Just one of the conventions we accepted when we migrated to Java 11 was to always use the `var` keyword unless it's a primitive Java type.